creme/creme_core/static/creme_core/js/search.js

```diff
@@ -77,9 +77,11 @@ creme.search.SearchBox = creme.component.Component.sub({
 
         this._element = element;
 
+        var _onInput = this._onInput.bind(this);
+
         this._input = element.find('input');
         this._input.on('focus', this._onShow.bind(this))
-                   .on('input paste', this.debounceDelay ? _.debounce(this._onInput.bind(this), this.debounceDelay) : this._onInput.bind(this))
+                   .on('input paste', this.debounceDelay ? _.debounce(_onInput, this.debounceDelay, false) : _onInput)
```

Comment:
Comme on disait, ptet

```suggestion
                   .on('input paste', this.debounceDelay ? _.debounce(_onInput, this.debounceDelay) : _onInput)
```